projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -1390,19 +1390,35 @@ function wpcom_is_edit_page_task_visible() {
 /**
  * Mark the customize_welcome_message task complete
  * if the subscription_options['invitation'] value
- * for the welcome message has changed.
+ * for the welcome message has changed on option update.
  *
  * @param string $old_value The old value of the welcome message.
  * @param string $value The new value of the welcome message.
  *
  * @return void
  */
-function wpcom_mark_customize_welcome_message_complete( $old_value, $value ) {
+function wpcom_mark_customize_welcome_message_complete_on_update( $old_value, $value ) {
 	if ( $value['invitation'] !== $old_value['invitation'] ) {
 		wpcom_mark_launchpad_task_complete( 'customize_welcome_message' );
 	}
 }
-add_action( 'update_option_subscription_options', 'wpcom_mark_customize_welcome_message_complete', 10, 3 );
+add_action( 'update_option_subscription_options', 'wpcom_mark_customize_welcome_message_complete_on_update', 10, 3 );
```

Comment:
Nit: We're only looking at the first two arguments.
```suggestion
add_action( 'update_option_subscription_options', 'wpcom_mark_customize_welcome_message_complete_on_update', 10, 2 );
```